imsim/stamp.py

```diff
@@ -493,7 +493,8 @@ def fix_seds(prof):
                                photon_ops=photon_ops,
                                rng=self.rng,
                                sensor=galsim.Sensor(),
-                               n_subsample=1
+                               n_subsample=1,
+                               maxN=int(1e6)
```

Comment:
I guess this is ok.  I think I'd rather it be a config parameter, so the user can change it, but this is how we currently do the other maxN (when method=phot), so this is fine for now.

Reply:
I thought about making it a config parameter, but I wasn't entirely sure where it should go.  I guess we should make both occurrences of `maxN` separate config parameters?  Anyway, I'd vote for fixing this later.

Reply:
I'd make just one maxN parameter that applies to both branches.  The reasons to change it would apply to both.

Reply:
If you're willing to do it now, this is the snippet I'd recommend (at line 477 before the method branching):
```
maxN = int(1e6)
if 'maxN' in config:
    maxN = galsim.config.ParseValue(config, 'maxN', base, int)[0]
```
Then in both calls to drawImage, use `maxN=maxN`.

Reply:
Thanks, I've implemented this.